practice_app/api/tests.py

```diff
@@ -19,7 +19,7 @@ def test_404_responses(self):
         self.assertEquals(self.c.get("/api/serp-api/title=sad").status_code, 404)
         self.assertEquals(self.c.get("/api/serp-api/?title=&").status_code, 404)
 
-    @skip('limited use for this api')
+    
```

Comment:
Don't remove this one. This test should be skipped.

Reply:
ok. i added it.